exercises/react/.meta/template.j2

```diff
@@ -0,0 +1,66 @@
+{%- import "generator_macros.j2" as macros with context -%}
+from functools import partial
+{{ macros.header(imports=['InputCell', 'ComputeCell']) }}
+{% set types = {'input': 'InputCell', 'compute': 'ComputeCell'} -%}
+
+class {{ exercise | camel_case }}Test(unittest.TestCase):
+    {% for case in cases -%}
+    {% set cells = case['input']['cells'] -%}
+    {% set operations = case['input']['operations'] -%}
+    def test_{{ case["description"] | to_snake }}(self):
+        {% if case["comments"] -%}
+        """
+        {% for comment in case["comments"] -%}
+        {{ comment }}
+        {% endfor -%}
+        """
+        {% endif -%}
```

Comment:
Skip the comments; these are notes for those implementing the exercise, not for those solving it.